src/main/java/com/wiley/autotest/selenium/elements/upgrade/TeasyWindow.java

```diff
@@ -65,4 +65,8 @@ public void waitForScriptsToLoad() {
     public String getUrl() {
         return driver.getCurrentUrl();
     }
+
+    public void scrollTo(TeasyElement element) {
```

Comment:
@mosadchiy as this scroll does no specify vertical/horizontal - we need to scroll both directions. Will current impl scroll horizontally (if needed)?

Reply:
Yes this method scrolls both directions in necessary